src/resources/Tutorials.js

```diff
@@ -1,16 +1,60 @@
-import React from "react";
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Youtube from "react-youtube";
 import "./resources.css";
 
+const fakeData = require("./fakeData");
+
+// tags is currently unused
+function ToggleItem({ id, title, description, url, tags }) {
+	const [showButton, setShowButton] = useState(false);
+
+	return (
+		<div className="resource-list" key={id}>
+			<button
+				type="button"
+				className={`resource-list-title ${
+					showButton && "resource-list-title-active"
+				}`}
+				onClick={() => setShowButton(prevState => !prevState)}
+			>
+				{title}
+			</button>
+
+			{showButton && (
+				<div className="resource-list-description">
+					{description}
+					<a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
Just wondering, is youtube video dropdown (not the url) will be in this commit?

Reply:
Yeah video dropdown isn't in this PR. Do you want me to implement it here too?

Reply:
I think it should be in separate PR.